docs/src/main/sphinx/release/release-390.md

```diff
@@ -0,0 +1,59 @@
+# Release 390 (13 July 2022)
+
+## General
+
+* Update minimum required Java version to 17.0.3. ({issue}`13014`)
+* Add support for [setting comments on views](/sql/comment). ({issue}`8349`) ({issue}`8349`)
+* Improve performance of queries with an `UNNEST` clause. ({issue}`10506`)
+* Fix potential query failure when spilling to disk is enabled by the
```

Comment:
maybe 

```suggestion
* Fix potential query failure when spilling to disk is enabled with the
```
```suggestion
* Fix potential query failure when spilling to disk is enabled by the
```

Reply:
I don't think that's necessarily any better/clearer